### DISCUSSION TITLE
Jhipster-Blueprint

### DISCUSSION THREAD

**Discussion by arunkumarArunachalam1** [jhipster/generator-jhipster#26602] [General]
I'd like to learn about creating and implementing a JHipster blueprint, as I'm new to this,
If online blogs aren't sufficient for understanding how to create and implement a JHipster blueprint. if any one has document or article share with me.

**Comment by DanielFran** [comment]
Hi @arunkumarArunachalam1 
You can start by reading the info from the site:
- [Blueprint Basics](https://www.jhipster.tech/modules/extending-and-customizing/)
- [Creating a blueprint](https://www.jhipster.tech/modules/creating-a-blueprint/)

**Comment by Meena1903** [comment]
Hello contributors, 
     I tried reading the sites recommended but I still cannot figure out the concept of jhipster blueprints do you have any simple demo blueprints that I could refer?

**Reply by Alchemik** [reply]
it's a bit old, but maybe it will still be useful to you: https://medium.com/@anth.viard/build-your-own-jhipster-with-modules-and-blueprints-dd7a2833a388

**Comment by Kosha-account** [comment]
The official documentation is a bit confusing, and even if I assume I succeed to decrypt it, but no luck to run my blueprint thus not sure if my side mistakes or docs outdated. I think it would be very helpful to all java hipsters
